src/builder.rs

```diff
@@ -1090,6 +1090,9 @@ fn build_with_store_internal(
 		// If we act as an LSPS2 service, we need to to be able to intercept HTLCs and forward the
 		// information to the service handler.
 		user_config.accept_intercept_htlcs = true;
+
+		// If we act as an LSPS2 service, we allow forwarding to unnannounced channels.
+		user_config.accept_forwards_to_priv_channels = true;
```

Comment:
Does it justify an addition to the `lsps2_client_service_integration` test for the non-jit case? Seems a lot of it is already in place in that test.